lectures/input_output.md

```diff
@@ -579,10 +573,11 @@ High ranking sectors within this measure are important buyers of intermediate go
 
 A demand shock in such sectors will cause a large impact on the whole production network.
 
-The following figure displays the output multipliers for the sectors represented in {numref}`us_15sectors`.
+The following figure displays the output multipliers for the sectors represented
+in {numref}`us_15sectors`.
 
 ```{code-cell} ipython3
-tags: [hide-input]
+:tags: [hide-cell]
```

Comment:
Hi @jstac,

I noticed that you changed `[hide-input]` to `[hide-cell]`. I reverted this because `[hide-cell]` not only hides the code cell but also hides the graph generated from the code : )
 
Please let me know if this needs further changes.

Reply:
Thanks @HumphreyYang -- I was tired so making mistakes :-)